spec/.eslintrc.js

```diff
@@ -2,5 +2,8 @@ module.exports = {
   env: {
     jasmine: true,
     atomtest: true
+  },
+  rules: {
```

Comment:
Note that there are several cases below that this will affect as the strings have `'` in them, just use `"foo 'bar'"`.